sklearn/datasets/tests/test_openml.py

```diff
@@ -1,7 +1,7 @@
 """Test the openml loader.
 """
 import gzip
-from io import BytesIO
```

Comment:
I would leave this import here.
Explanation: Fist, why the change at all? Second: There is not strict convention. Usually, we start with imports from the standard library of which `io` is a part.